doc/zconfig.rst

```diff
@@ -0,0 +1,1989 @@
+=========
+ ZConfig
+=========
+
+
+This document describes the syntax and API used in configuration files
+for components of a Zope installation written by Zope Corporation.  This
+configuration mechanism is itself configured using a schema specification
+written in XML.
+
+.. toctree
+
+
+.. _intro:
+
+Introduction
+============
+
+Zope uses a common syntax and API for configuration files designed for
+software components written by Zope Corporation.  Third-party software
+which is also part of a Zope installation may use a different syntax,
+though any software is welcome to use the syntax used by Zope
+Corporation.  Any software written in Python is free to use the
+:mod:`ZConfig` software to load such configuration files in order to
+ensure compatibility.  This software is covered by the Zope Public
+License, version 2.1.
+
+The :mod:`ZConfig` package has been tested with Python 2.3.  Older
+versions of Python are not supported.
```

Comment:
I hope that next PR will simply remove this paragraph; this is better handled with Trove classifiers in the packaging.